client.go

```diff
@@ -173,6 +178,11 @@ type Client struct {
 // Construct and return a new client Conn, setting default config
 // values as necessary.
 func newClient(config Config, hosts []string) *Client {
+	if config.DialContext == nil {
```

Comment:
Don't you need to default config.Timeout before defaulting this?

Reply:
Yep.

Reply:
Done (719b580). I also noted where the library should wire up a parent `Context` in the future to allow overall FTP operations to be governed by a caller’s `Context`.